plio/io/io_controlnetwork.py

```diff
@@ -302,7 +244,7 @@ def _set_pid(pointid):
                         point_spec.aprioriCovar.extend(arr)
                     else:
                         setattr(point_spec, attr, attrtype(g.iloc[0][attr]))
-            point_spec.type = 2  # Hardcoded to free this is bad
+            # point_spec.type = 2  # Hardcoded to free this is bad
```

Comment:
Is this something that needs to be set, or do we only care about propagating the measure types? It might be set somewhere else and I'm just missing it

Reply:
Rather just propagate the point type which is done on the loop below here. It's easier to just change the point type in the input dataframe.